README.md

```diff
@@ -152,7 +152,7 @@ Attributes
 
 ### Kafka Connect
 * `node["confluent"]["kafka-connect"]["jar_urls"]` : an array of urls to remote files to download and install in the directory `share/java/kafka-connect-all` located in the extracted confluent directory which is where connect looks by default.
-* `node["confluent"]["kafka-connect"]["properties_files"]` : a hash where the key is a property file name, and the value is a hash of keys/values for the property file. Used to drop in property files via chef config as opposed to the rest api.
+* `node["confluent"]["kafka-connect"]["properties_files"]` : a hash where the key is a property file name, and the value is a hash of keys/values for the property file. Used to drop in property files (connector configuration for standalone mode) via chef config as opposed to the rest api.
```

Comment:
"and uses the keys as the startup files for standalone mode"? Not sure how explicit you want to be